src/topp/FileInfo.cpp

```diff
@@ -848,6 +848,21 @@ class TOPPFileInfo : public TOPPBase
       os << "\nFor pepXML files, only validation against the XML schema is implemented at this point."
          << '\n';
     }
+    else if (in_type == FileTypes::MZTAB)
+    {
+      MzTab mztab;
+      MzTabFile().load(in, mztab);
+      os << "mzTab-version: " << mztab.getMetaData().mz_tab_version.get() << '\n'
+         << "mzTab-mode: " << mztab.getMetaData().mz_tab_mode.get() << '\n'
+         << "mzTab-type: " << mztab.getMetaData().mz_tab_type.get() << '\n'
+         << "number of PSMs: " << mztab.getPSMSectionRows().size() << '\n'
+         << "number of peptides: " << mztab.getPeptideSectionRows().size() << '\n'
```

Comment:
Strictly speaking I think the number of rows does not correspond to the number of peptides / PSMs. This could be a bit misleading

Reply:
Ahh right. I keep forgetting this weird design decision...
Thanks for catching this.
Fixed by taking duplicated PSM's into account (does only apply to PSM, not peptides according to mzTab spec doc).